content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md

```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
 
 ### Create a ticket in the MyRackspace portal
 
-1. Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with your username and
+**Step 1.** Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with your username and
    password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+<img width="409" alt="Enter credentials for Username and Password" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk4.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+**Step 2.** In the top navigation bar, choose **Support** > **Support Center**.
 
-4. Select the **Subject** field.
+<img width="941" alt="Select support center from top navigation bar" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk5.png">
 
-5. From the drop-down menu, select **Add Disk Space to Server or Hypervisor.**
+**Step 3.** From the **Common Request** menu select **Add Disk Space to Sever or Hypervisor** as shown in following image.
 
-   {{<image src="adddisk1.png" alt="" title="">}}
+<img width="677" alt="Select Add Disk Space to Server or Hypervisor from common request template" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk6.png">
 
-6. In the **Ticket Details** section, select a device from the drop-down menu.
+**Step 4.** In **Issue Details** section, select a device from the drop-down menu.
 
-7. Enter the amount of disk space needed, and choose whether to add the space to an existing volume or to
-   configure as a new drive.
+<img width="440" alt="Select required device from drop-down menu" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk8.png">
```

Comment:
This image shows real device number, and names.
Did you alter the information before taking the screenshot?

Reply:
Replace image as suggested.